docs/website/docs/building-from-source/getting-started.md

```diff
@@ -11,19 +11,25 @@ compiler:
 
     <!-- TODO(scotttodd): annotation about gcc vs clang -->
 
-    ``` shell
-    sudo apt install cmake clang
```

Comment:
I think it's worth having the copy paste commands here for cmake/ninja and if we are going to say clang below then we should have that too. I'd really like a CI for gcc though so we didn't have to specify.
```shell
python -m pip install ninja
sudo apt install cmake clang lld
```

Reply:
Added back. Agreed on the CI. But I'd also like fewer users to show up in forums having gone down a rabbit hole on this.